game-core/src/main/java/tools/map/making/MapPropertiesMaker.java

```diff
@@ -320,6 +320,8 @@ private void loadProperties() {
       try (InputStream in = new FileInputStream(centerName)) {
         final Properties properties = new Properties();
         properties.load(in);
+        System.out.println("Successfully loaded Properties File:");
+        properties.list(System.out);
```

Comment:
Did you intend to leave this system.out?

Reply:
Yes, the code looks like it just checks if the properties file is valid, but the static analysis indicated that the properties files content is never actually used.
That's why I printed the content, just to make use of it.

Reply:
Looking at the code this feature is dead anyways.
A JMenuItem entry is being created but never added to any active Window.
Removing...